README.md

```diff
@@ -94,8 +152,39 @@ end
 end
 ```
 
-If you are not using ActiveJob, the same approach may be used with minor
-adjustments.
+Now, if you run `rails db:migrate`, `rails db:schema:load` or `rails
+db:schedule_jobs` all Jobs inheriting from `CronJob` are scheduled.
+
+*If you are not using ActiveJob, the same approach may be used with minor
+adjustments.*
+
+### Cheap "health"-display of schedules
+
+If you have an admin controller or some place where you want to show whether or
+not all `CronJob`s are scheduled as expected, you can add the following snippets
+to your controller to show a flash if some schedule is missing. You can also
+expose a controller action to trigger `ScheduledJobCheck.schedule_all!` .
+
+```ruby
+# app/controller/admin_controller.rb
+# ...
+  def index
+    if !ScheduledJobCheck.all_scheduled?
+      flash.now[:error] = t('.problem_not_all_jobs_scheduled')
+    end
+    # ... whatever AdminController does, e.g.
+    @jobs = Delayed::Job.all
+    # ...
+  end
+# ...
+```
```

Comment:
This example is too special and does not add very much value IMHO. I guess everybody who'd like to implement something like this would easily figure that out. I'd strip it completely.

Reply:
see above. Give me a nod and I'll remove it.

Reply:
Yes, please remove it.